.travis.yml

```diff
@@ -1,28 +1,14 @@
-language: go
-sudo: false
-go:
-  - 1.7
-  - 1.8
-  - tip
-
-go_import_path: go.uber.org/fx
-
+sudo: required
+services:
+  - docker
 env:
-  - V=1 COVERMODE=atomic
-
+  - DOCKER_GO_VERSION=1.7 V=1 COVERMODE=atomic
+  - DOCKER_GO_VERSION=1.8 V=1 COVERMODE=atomic
```

Comment:
Personally this not only doesn't bother me, but I rather not be needing to test against tip - it's not like the one time every 6 months that golang is released we can't re-run `make dockerrun` :)